src/griddyn/measurement/gridGrabbers.cpp

```diff
@@ -85,17 +85,6 @@ void gridGrabber::updateField (const std::string &fld)
     }
 
     loaded = checkIfLoaded ();
-    if (loaded)
```

Comment:
Why were these sections removed?

Reply:
The only way to get desc is gridGrabber::getDesc, which did makeDescription/loaded checks already. The customDesc check is already there too